application/controllers/Api.php

```diff
@@ -1792,4 +1792,131 @@ private function validateDate($date, $format = 'Y-m-d')
         return $d && $d->format($format) === $date;
     }
     
+	/**
+     * Get one node of the organization
+     * @author Mickael ROMMME
+     */
+    public function organization_node($id) {
+        if (!$this->server->verifyResourceRequest(OAuth2\Request::createFromGlobals())) {
+            $this->server->getResponse()->send();
+        } else {
+            $this->load->model('Organization_model');
+            $result = $this->Organization_model->getName($id);
+            echo json_encode($result);
```

Comment:
The MIMETYPE (output encoding) is not set. You should rely on the framework, e.g.:

                $this->output
                    ->set_content_type('application/json')
                    ->set_output(json_encode($result));